scripts/create_spec_repo/Package.swift

```diff
@@ -20,6 +20,9 @@ import PackageDescription
 
 let package = Package(
   name: "PodSpecBuilder",
+  platforms: [
+    .macOS(.v10_13),
```

Comment:
Why this change?

Reply:
This is due to the [Process/Shell update](https://github.com/firebase/firebase-ios-sdk/pull/9809/files#diff-56d9e831e359a173355a51a81b46b99cd8282fdab7b615b1f34c66c12637a3eaL93-R101) in the change.
[Launch](https://developer.apple.com/documentation/foundation/process/1414189-launch) and [LaunchPath](https://developer.apple.com/documentation/foundation/process/1414221-launchpath) in  [Process](https://developer.apple.com/documentation/foundation/process))

Reply:
Considering the two previous APIs were deprecated and the tool is mainly running on GHA, which has a consistent version above 10.13, I set the version of the executable instead of using `#if available`.